public/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>DPdash</title>
+  </head>
+
+  <body>
+    <div id="root"></div>
+    <script src="./bundle.js" />
```

Comment:
```suggestion
    <script src="./bundle.js"></script>
```

Reply:
😕  unsure what is the difference

Reply:
one is proper html and the other isn't 😄